pom.xml

```diff
@@ -25,7 +25,7 @@
         <changelist>-SNAPSHOT</changelist>
         <hpi.compatibleSinceVersion>2.2.0</hpi.compatibleSinceVersion>
         <java.level>8</java.level>
-        <jenkins.version>2.176.4</jenkins.version>
+        <jenkins.version>2.235.1</jenkins.version>
```

Comment:
I think we require https://github.com/jenkinsci/jenkins/pull/4603 here.

Reply:
This means this change won't be viable for quite a while yet.

Reply:
A matter of opinion I suppose. Anyone running an older version of Jenkins is susceptible to disclosed vulnerabilities so why should they expect to get miscellaneous bug fixes and improvements either?